frontend/src/stores/drag-drop.store.ts

```
@@ -0,0 +1,201 @@
+        const isCompatible = target.acceptedTypes.some((type) => compatibleTypes.includes(type))
+        if (!isCompatible) return false
+
+        // Check nullable constraints
+        if (target.isRequired && column.nullable) return false
+
+        // Additional validation based on target type
+        switch (target.type) {
+          case 'label':
+          case 'alias':
+            // Check for reasonable length constraints
+            const maxLength = target.type === 'label' ? 250 : 100
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)